package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "lodash": "^4.17.4",
     "next-frame": "^0.2.3",
-    "react-native-meteor": "^1.0.3",
+    "react-native-meteor": "https://github.com/lc3t35/react-native-meteor#dev",
```

Comment:
Why this branch?

Reply:
Oops ! Sorry I didn't noticed it was pushed to the upstream by VisualStudioCode, it's a dev only branch. I was trying vforvasile/rn-meteor, but it's missing exports (such as connection), so I'm testing ....
You can get rid of this, I'll pay more attention next time.